docs/about/rationals.md

```diff
@@ -0,0 +1,75 @@
+# Rationals
+
+Before we recommend anything it goes through a process of evaluations.
+In short
+
+1. We only recommend usage of a features/function if it is release in a stable version of a least ONE browser
+2. We only recommend libraries if they offer an es module version
+
+::: warning
+We have ONE exception to this rule and that is 'bare modules'.
+This is such a powerful an prominent feature in the current js ecosystem so if you don't use it you basically need to implement everything yourself.
```

Comment:
powerful an prominent -> powerful and prominent

Reply:
thx for taking a look 🤗 - unfortunately, this paragraph changed 🙈